pages/studentAssessmentExam/studentAssessmentExam.js

```diff
@@ -55,6 +76,33 @@ router.post('/', function (req, res, next) {
         res.redirect(res.locals.urlPrefix + '/assessment_instance/' + assessment_instance_id);
       }
     );
+  } else if (req.body.__action === 'join_group') {
+    const joinCode = req.body.join_code;
+    groupAssessmentHelper.joinGroup(joinCode, res, function (err, joinErr) {
+      if (ERR(err, next)) return;
+      if (joinErr) {
+        //display error
```

Comment:
```suggestion
```

Reply:
Done in [6be3ef3](https://github.com/PrairieLearn/PrairieLearn/pull/5290/commits/6be3ef357b4e3ccdfc6476814096629c6fbb1208)